experiments/clang_interface/Makefile

```diff
@@ -0,0 +1,70 @@
+
+all: clang_interface clang_interface-release
+
+clang_interface: clang_interface.cc Makefile
+	clang++80 clang_interface.cc -o clang_interface \
```

Comment:
Better to use "clang++", we don't want that to break when we upgrade. Let the environment control which version we use.

Reply:
I'm tempted to just remove this Makefile.  It works on my system (clang++ won't, I think, because it's too old to support C++17), but it hard codes things like the locations of my LLVM builds.  It's mostly there as an example: I don't expect anyone to actually use this code as-is.